src/toolbox.ts

```diff
@@ -1922,9 +1920,14 @@ export class KeypointSliderItem extends ToolboxItem {
      * 
      * @param ulabel ULabel object
      * @param filter_value The number between 0-100 which annotation's confidence is compared against
+     * @param redraw whether or not to redraw the annotations after filtering
      * @returns Annotations that were modified, organized by subtask key
      */
-    private filter_annotations(ulabel: ULabel, filter_value: number): {[key: string]: string[]} {
+    private filter_annotations(ulabel: ULabel, filter_value: number = null, redraw: boolean = false): void {
+        if (filter_value === null) {
+            // Use stored filter value if none is passed in
+            filter_value = Math.round(this.filter_value * 100);
```

Comment:
this fixes the bug where filtering on load wouldn't work as expected, it was using a filter value 100 times smaller than it should've